flink/v1.17/build.gradle

```diff
@@ -97,7 +107,7 @@ project(":iceberg-flink:iceberg-flink-${flinkMajorVersion}") {
       exclude group: 'com.google.code.findbugs', module: 'jsr305'
     }
 
-    testImplementation("org.apache.hive:hive-metastore") {
+    testImplementation(libs.hive2.metastore) {
```

Comment:
L126 needs to be changed to `testImplementation libs.awaitility`. Same for the other Flink versions

Reply:
done